app/dashboard/static/js/app/view-socs-soc-job-kernel.2017.4.js

```diff
@@ -20,380 +23,36 @@
-        html.replaceContent(
-            document.getElementById('accordion-container'),
-            html.errorDiv('Error loading data.'));
-        html.replaceByClassHTML('loading-content', '&infin;');
-    }
-
-    function getBootsDone(response) {
-        var bootPanel,
-            failButton,
-            results;
-
-        results = response.result;
-        if (results.length === 0) {
-            html.replaceContent(
-                document.getElementById('accordion-container'),
-                html.errorDiv('No data found.'));
-        } else {
-            bootPanel = bootView(results, gFileServer).draw();
-
-            if (gSearchFilter && gSearchFilter.length > 0) {
-                switch (gSearchFilter) {
-                    case 'fail':
-                        document.getElementById('#fail-cell').click();
-                        break;
-                    case 'success':
-                        document.getElementById('#success-cell').click();
-                        break;
-                    case 'unknown':
-                        document.getElementById('#unknown-cell').click();
-                        break;
-                }
-            } else if (!loadSavedSession()) {
-                if (bootPanel.hasFail) {
-                    // If there is no saved session, show only the failed ones.
-                    Array.prototype.forEach.call(
-                        document.getElementsByClassName('df-failed'),
-                        function(element) {
-                            element.style.setProperty('display', 'block');
-                        }
-                    );
-                    Array.prototype.forEach.call(
-                        document.getElementsByClassName('df-success'),
-                        function(element) {
-                            element.style.setProperty('display', 'none');
-                        }
-                    );
-                    Array.prototype.forEach.call(
-                        document.getElementsByClassName('df-unknown'),
-                        function(element) {
-                            element.style.setProperty('display', 'none');
-                        }
-                    );
-
-                    failButton = document.getElementById('fail-btn');
-                    Array.prototype.forEach.call(
-                        failButton.parentElement.children, function(element) {
-                            if (element === failButton) {
-                                html.addClass(element, 'active');
-                            } else {
-                                html.removeClass(element, 'active');
-                            }
-                        }
-                    );
-                } else {
-                    html.addClass(
-                        document.getElementById('all-btn'), 'active');
-                }
-            }
-        }
-    }
-
     function updateDetails(response) {
```

Comment:
I think the argument `response` should be renamed `results` to avoid ambiguity.  The results are taken from the response.